projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-comment-endpoint.php

```diff
@@ -71,7 +71,13 @@ public function get_comment( $comment_id, $context ) {
 			return new WP_Error( 'unknown_comment', 'Unknown comment', 404 );
 		}
 
-		$types = array( '', 'comment', 'pingback', 'trackback', 'review' );
+		/**
+		 * Filter the comment types that are allowed to be returned.
+		 *
```

Comment:
Let's add the version when the filter was added, as well as the module name so the filter is properly categorized in the Codex:

```suggestion
		 *
		 * @since $$next-version$$
		 *
		 * @module json-api
		 *
```